src/drivers/intel/cavs/mn.c

```diff
@@ -19,6 +19,11 @@
 #define tracev_mn(__e, ...) \
 	tracev_event(TRACE_CLASS_MN, __e, ##__VA_ARGS__)
 
+enum mclk_source {
+	MN_MCLK_SOURCE_NONE = 0,
+	MN_MCLK_SOURCE_MN,
```

Comment:
I'm trying to understand this but what does MN_MCLK_SOURCE_MN really stand for?

Reply:
That MCLK is driven by clock coming from M/N. I think it can be just bool, because we have only 1 possible source atm (M/N), it makes sense to have it like this for BCLK and I did it also for MCLK to keep logic similar, but I understand it may cause more confusion.

Reply:
Changed to bool table and changed description + added some desc to bclk enum